train.py

```diff
@@ -15,12 +15,16 @@
 from model import make_model
 from modules.label_smoothing import LabelSmoothing
 from data.dataset import MyTokenizedMidiDataset, load_cache_dataset
-from utils import vocab_sizes, learning_rate_schedule, calculate_average_distance
+from utils import learning_rate_schedule, calculate_average_distance
 
 
 @hydra.main(version_base=None, config_path="configs", config_name="main")
 def main(cfg: DictConfig):
-    train_dataset, val_dataset = load_datasets(dataset_name=cfg.dataset_name, dataset_cfg=cfg.dataset)
+    train_dataset, val_dataset = load_datasets(
+        dataset_name=cfg.dataset_name,
+        dataset_cfg=cfg.dataset,
+        predict=cfg.predict,
+    )
     train_model(train_dataset, val_dataset, cfg)
```

Comment:
I am struggling to think of a better way of passing the argument about the name of a column we want to predict and number of bins (always 200 right now) down to tgt_encoder in load_cache_dataset.